articles/quickstart-microsoft-provider-format.md

```diff
@@ -279,41 +279,86 @@ except ImportError:
 
 #### Create a quantum register
 
-1. First, you create a 'devices' object to import the PASQAL quantum computer target, Fresnel. [Fresnel QPU](xref:microsoft.quantum.providers.pasqal#fresnel) provides predefined trap layouts that you can utilize to create your quantum register. To configure your quantum registers, you arrange an array of qubits. 
+You need to define both a register and a layout before proceeding. The register specify where atoms will be arranged, while the layout specifies the positioning of traps necessary to capture and structure these atoms within the register. 
+
+For details on layouts, see the [Pulser documentation](https://pulser.readthedocs.io/en/stable/tutorials/reg_layouts.html).
+
+- First, you create a 'devices' object to import the PASQAL quantum computer target, [Fresnel](xref:microsoft.quantum.providers.pasqal#fresnel).
 
     ```python
     from pulser_pasqal import PasqalCloud
 
     devices = PasqalCloud().fetch_available_devices()
     QPU = devices["FRESNEL"]
-
-    # List all available calibrated register layouts
-    for calibrated_register_layout in QPU.calibrated_register_layouts.keys():
-        print(calibrated_register_layout)
     ```
 
-1. Next, you define the layout for your qubit register. In this example, you use a `TriangularLatticeLayout(61, 5.0µm)` as the trap layout. 
+##### Pre-calibrated layouts
+
+The device defines a list of [pre-calibrated layouts](https://pulser.readthedocs.io/en/stable/tutorials/reg_layouts.html#Devices-with-pre-calibrated-layouts). You can build your register out of one of these layouts.
+
+This is the recommended option because it will improve the performance of the QPU
+
+* Option 1: Define your register using pre-calibrated layouts:
```

Comment:
```suggestion
This is the recommended option because it will improve the performance of the QPU.

* Option 1: Define your register using pre-calibrated layouts
```